app/views/account/registrations/edit.html.erb

```diff
@@ -36,3 +36,29 @@
     </div>
   </div>
 </div>
+<div class='row'>
+  <div class='col-sm-12'>
+    <div class='card'>
+      <div class='card-header'>
+        <%= fa_icon 'phone' %>
+        <strong><%= t('.mobile_phone_updating') %></strong>
+      </div>
+      <div class='card-body'>
+        <%= simple_form_for(resource, url: { action: :phone_verification, controller: 'account/registrations' }, remote: true, html: { method: :put }) do |f| %>
+          <div class='row'>
+            <div class='form-group col-sm-12'>
+              <%= f.error_notification %>
+            </div>
+            <div class='form-group col-sm-12'>
+              <%= f.input :mobile_phone, autofocus: true %>
```

Comment:
Ülke seçimini eklememiz gerekiyor diğer türlü telefon numarasını ülke koduyla yazmak gerekiyor.

Reply:
Bu yapılmadıysa yeni issue açalım (PR'ı bekletmemek için).

Reply:
[issue](https://github.com/omu/nokul/issues/940) açıldı.